pilot/pkg/security/authz/policy/test/helper.go

```diff
@@ -12,7 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package policy
+package test
```

Comment:
Instead of changing to a different package, can we just remove the import of "testing" in this package and replace the `t *testing.T` to a function of type `fatalFn func(format string, args ...interface{})`, and change the calling site to pass t.Fatal?

(Or you can remove the use of the testing package completely, I think that's fine too)